### DISCUSSION TITLE
Support for NVIDIA Jetson AGX Xavier 32 GB

### DISCUSSION THREAD

**Discussion by Martin-Bohring** [OE4T/meta-tegra#711] [Q&A] [answered]
Hello,
I have not seen the production version of the NVIDIA Jetson AGX Xavier 32 GB in the list of supported NNIDIA SoM.

Just to double check, here my current assumptions:

- We start our development on the NVIDIA Developer Board
- Bring up Yocto + OE4T on that board using to get the SW Developers started
- Develop Custom Carrier Board with additional Custom HW (cannot disclose here what exactly)
- Create custom Yocto BSP Layer for the additional Custom HW
- This BSP layer can build upon / extend the OE4T Layer
- Custom Carrier Board + NVIDIA Jetson AGX Xavier 32 GB Production Version + OE4T + Custom Yocto BSP = Running System?

Being new to OE4T it is not clear to me if my assumptions are completely true.

Especially the Boards Supported List mentions the "Jetson AGX Xavier development kit" but not Jetson AGX Xavier on another Carrier board:

Can somebody please confirm that Jetson AGX Xavier is **supported** on a **Custom Carrier Board**?
(And yes the Custom part will come from somewhere else and not from OE4T)

**Comment by madisongh** [comment, marked as answer]
We don't have machine configs set up for individual SOMs, since you can't boot them without some kind of carrier.  The AGX Xavier SOM in a custom carrier should work just fine, though - just use the `jetson-agx-xavier-devkit.conf` as a starting point for your customized system.

There are some notes [on the wiki](https://github.com/OE4T/meta-tegra/wiki/Creating-a-custom-MACHINE) about other customization steps.
